app/views/expenses/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<section>
+  <h2 class="text-center">Expenses</h2>
+  <div class="text-secondary text-center">Group - <%= @group.name %></div>
+  <p>Total price: $<%= @group.total_price %></p>
+  <% if @expenses.blank? %>
+    <p>No Expenses found.</p>
+  <% else %>
+    <%= render 'list' %>
+  <% end %>
+  <%= link_to 'Add a new Expense', new_expense_path, class: 'btn bg-sec w-100 text-light' %>
```

Comment:
- [ ] Please make sure this button looks similar to that provided in the original design. It should be at the bottom of the page with a thin white border surrounding the test like I mentioned in the previous comment